types/types.go

```diff
@@ -142,6 +142,7 @@ type Container struct {
 	SizeRw     int64 `json:",omitempty"`
 	SizeRootFs int64 `json:",omitempty"`
 	Labels     map[string]string
+	State      string
 	Status     string
```

Comment:
true :stuck_out_tongue_closed_eyes: But I thought it made more sense to make it alphabetically (but Labels is not so... meh.. :smiley:)